project/AkkaBuild.scala

```diff
@@ -79,12 +79,9 @@ object AkkaBuild extends Build {
         archivesPathFinder.get.map(file => (file -> ("akka/" + file.getName)))
       },
       Maven.projects in Maven.mvn := Seq(file("akka-samples") / "akka-docs-java-lambda"),
-      // FIXME when jenkins issue is solved, re-enable pr validation of:
-      //       Maven.mvnExec
-      //       test in Test in httpJava8Tests
       validatePullRequest <<= Seq(test in Test in stream, SphinxSupport.generate in Sphinx in docsDev,
-      test in Test in streamTestkit, test in Test in streamTests, test in Test in streamTck,
-      test in Test in httpCore, test in Test in http, test in Test in httpJavaTests,
+      Maven.mvnExec, test in Test in streamTestkit, test in Test in streamTests, test in Test in streamTck,
```

Comment:
what are we actually running with maven, and why? can that be run in the same way as httpJava8Tests instead?